test/integration/test_spark_lightning.py

```diff
@@ -782,103 +782,7 @@ def test_early_stop_callback(self):
                 assert pred.dtype == torch.float32
 
     """
-    Test train model with pytorch data loader
-    """
-    def test_train_with_pytorch_data_loader(self):
```

Comment:
Please add equivalent test cases with the new data_module api.

Reply:
The added tests should be used to test datamodule parameter, instead of covering all customized data loaders.